### ISSUE TITLE
.oh-my-zsh/lib/misc.zsh:3: parse error near `then' : After update 

### ISSUE THREAD

**Issue by vijayrawatsan** [robbyrussell/oh-my-zsh#4022]
## Load smart urls if available
for d in $fpath; do
	if [[ -e "$d/url-quote-magic"]]; then
		autoload -U url-quote-magic
		zle -N self-insert url-quote-magic
	fi
done

## jobs
setopt long_list_jobs

## pager
export PAGER="less"
export LESS="-R"

## super user alias
alias _='sudo'
alias please='sudo'

## more intelligent acking for ubuntu users
alias afind='ack-grep -il'

# only define LC_CTYPE if undefined
if [[ -z "$LC_CTYPE" && -z "$LC_ALL" ]]; then
	export LC_CTYPE=${LANG%%:*} # pick the first entry from LANG
fi

# recognize comments
setopt interactivecomments

**Comment by leepa** [comment]
:+1: although this issue is badly formatted... yes this is an issue.

**Comment by skyjia** [comment]
+1

**Comment by atipugin** [comment]
+1

**Comment by ChrisDevo** [comment]
+1

**Comment by ungarida** [comment]
+1

**Comment by drakmail** [comment]
+1

**Comment by imran-uk** [comment]
Fix with an extra space:

```
diff --git a/lib/misc.zsh b/lib/misc.zsh
index e908364..6d1a64e 100644
--- a/lib/misc.zsh
+++ b/lib/misc.zsh
@@ -1,6 +1,6 @@
 ## Load smart urls if available
 for d in $fpath; do
-       if [[ -e "$d/url-quote-magic"]]; then
+       if [[ -e "$d/url-quote-magic" ]]; then
                autoload -U url-quote-magic
                zle -N self-insert url-quote-magic
        fi
```

**Comment by drakmail** [comment]
Works for me

**Comment by Kevin-Lee** [comment]
The PR including the fix for this issue was created about 5 hours ago. #4002 We are all too late. :wink:

**Comment by haoxun** [comment]
+1

**Comment by agirorn** [comment]
+1

**Comment by 3lvis** [comment]
+1

**Comment by alexandrevicenzi** [comment]
Same here :+1:

**Comment by santiagobasulto** [comment]
+1

**Comment by fkumro** [comment]
+1

**Comment by idoo** [comment]
+1

**Comment by chrfis** [comment]
+1

**Comment by guillaumerose** [comment]
+1

**Comment by Donov4n** [comment]
+1

**Comment by adamantivm** [comment]
+1

**Comment by ewilazarus** [comment]
+1

**Comment by ncanceill** [comment]
Thanks everybody for reporting.

This was originally reported and #3986 and will be fixed once #4002 is merged.
